### ISSUE TITLE
Animepahe connection error

### ISSUE THREAD

**Issue by Default0X** [KevCui/animepahe-dl#37]
I tried using the tool and it worked fine but when I tried again, it gave the error:
curl: (7) Failed to connect to animepahe.com port 443 after 118 ms: Bad access
[ERROR] Anime slug not found!

I have installed all the dependencies and also have fast internet.

**Comment by KevCui** [comment]
Hi @Default0X, the error message looks a network connection issue or a server issue. Maybe try it again later? 🤞

**Comment by Default0X** [comment]
Thanks @KevCui . I'll give it some time. Although it's been doing this for some days now

**Comment by KevCui** [comment]
@Default0X oh, it's been for some days... :thinking: it shouldn't if it's network issue. Did you update some firewall configurations recently on your machine? If you try this command `curl -I https://animepahe.com`, will you get the connection established?

**Comment by Default0X** [comment]
Really appreciate your reply but it seems that that doesn't work either
![Error](https://user-images.githubusercontent.com/56763064/133926014-90bcb771-e209-4db2-815f-c88a29fd1f75.png)

**Comment by KevCui** [comment]
@Default0X Ok, curl the site directly is not working either. I don't know much about how to debug curl. Maybe you could search this curl error online and find some solutions.

**Comment by Default0X** [comment]
Alright @KevCui . I'll look it up and see what I can find

**Comment by Default0X** [comment]
So @KevCui , after many attempts at trying to fix the issue, I found out that the issue originated from the script restrictions that were enabled on my machine. Windows, on some machines, restricts scripts that it thinks are harmful from running.
So I fixed this by
1. Running cmd/PowerShell as admin
2. Enter the command: Get-ExecutionPolicy -List
3. Enter the command: Set-ExecutionPolicy Unrestricted
4. Enter "Y" to accept
5. Set-ExecutionPolicy Unrestricted -Force

That should be all. If this doesn't work, restart and try again.
-Cheers
